web/css/main.css

```
@@ -213,74 +339,1225 @@ body.pc-vertical-stretch {
+    @apply inline-flex;
+    @apply items-center;
+    @apply justify-center;
+    @apply shrink-0;
+    @apply w-4;
+    @apply h-4;
+    @apply rounded-full;
+    @apply border;
+    @apply border-solid;
+    @apply border-pc-grey-950;
+    @apply text-pc-grey-950;
+    font-family: Helvetica, Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)